processes/wg_offboarding.md

```diff
@@ -18,13 +18,9 @@ At any time until the last time mark any member can chime in and request to reta
 The following is a check-list of actions to be taken upon departure of users from the Security WG (either voluntarily or due to inactivity as described above):
 * [ ] Remove user from [Repo README](https://github.com/nodejs/security-wg/blob/master/README.md), [Triage Team](https://github.com/nodejs/security-wg/blob/master/processes/third_party_vuln_process.md)
 * [ ] Remove membership from [Node.js WG GitHub Team](https://github.com/orgs/nodejs/teams/security-wg)
-* [ ] Remove user from [HackerOne platform](https://hackerone.com/nodejs-ecosystem)
-* [ ] Revoke any user-specific access tokens from HackerOne platform
```

Comment:
I don't have insight into if there's any preexisting access that would need to be revoked.